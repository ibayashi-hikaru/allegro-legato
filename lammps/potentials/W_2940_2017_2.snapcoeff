# DATE: 2017-02-20 UNITS: metal CONTRIBUTOR: Mitchell Wood mitwood@sandia.gov CITATION: Wood, M. A. and Thompson, A. P. "Quantum-Accurate Molecular Dynamics Potential for Tungsten" arXiv:1702.07042 [physics.comp-ph]
# 
# LAMMPS SNAP coefficients for W

1 56
W 0.5 1
  0.781170857801
 -0.001794941735
 -0.016628679036
 -0.066625537037
 -0.073716343967
 -0.062913923923
  0.032552694672
 -0.134901744419
 -0.075076334103
 -0.148558616547
 -0.140808831101
 -0.166749145704
 -0.047487675984
 -0.049892090603
 -0.032483739965
 -0.114766534860
 -0.106759718242
 -0.125894850485
 -0.103409735225
 -0.095247335447
 -0.061998736346
 -0.053895610976
 -0.010799734206
 -0.011644828900
 -0.028316826924
  0.011176085541
  0.064619474684
 -0.023886279996
 -0.004099224312
 -0.056084222496
 -0.035551497650
 -0.056678501024
 -0.004905851656
 -0.015701146162
 -0.008462280779
  0.016429018676
  0.032432633993
 -0.010805361272
 -0.014841893457
  0.019414134562
 -0.008112452759
 -0.002700775447
  0.007032887063
 -0.009706065042
  0.008385967833
  0.028606085876
 -0.007003591067
  0.006467260152
 -0.006666986361
  0.029243285316
  0.002477673872
 -0.000199497504
  0.004068954075
  0.006036129972
 -0.013010633924
 -0.008314173699
