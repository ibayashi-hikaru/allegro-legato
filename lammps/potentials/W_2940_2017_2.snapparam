# DATE: 2017-02-20 UNITS: metal CONTRIBUTOR: Mitchell Wood mitwood@sandia.gov CITATION: Wood, M. A. and Thompson, A. P. "Quantum-Accurate Molecular Dynamics Potential for Tungsten" arXiv:1702.07042 [physics.comp-ph]
# 
# required
rcutfac 4.73442
twojmax 8

# optional

rfac0 0.99363
rmin0 0
bzeroflag 0
quadraticflag 0
