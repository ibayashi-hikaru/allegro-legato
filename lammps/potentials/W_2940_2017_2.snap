# DATE: 2017-02-20 UNITS: metal CONTRIBUTOR: Mitchell Wood mitwood@sandia.gov CITATION: Wood, M. A. and Thompson, A. P. "Quantum-Accurate Molecular Dynamics Potential for Tungsten" arXiv:1702.07042 [physics.comp-ph]
#
# Definition of SNAP+ZBL potential.
variable zblcutinner equal 4
variable zblcutouter equal 4.8
variable zblz equal 74

# Specify hybrid with SNAP and ZBL

pair_style hybrid/overlay &
zbl ${zblcutinner} ${zblcutouter} &
snap
pair_coeff 1 1 zbl ${zblz} ${zblz}
pair_coeff * * snap W_2940_2017_2.snapcoeff W_2940_2017_2.snapparam W

#Nomenclature on the snap files are Element_DakotaID_Year_Month
